test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json

```diff
@@ -3,6 +3,6 @@
   "cloudrail": "yes",
   "kics": "no",
   "snyk": "no",
-  "terrascan": "no",
+  "terrascan": "yes",
```

Comment:
In the output below I can't see DocDB cluster encryption violation

Reply:
@yi2020 sure, I will update this.

Reply:
done, made the changes